pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes Konvoy 1.8.4
+title: Release Notes Konvoy 1.8.4
+menuWeight: 40
+excerpt: View release-specific information for Konvoy 1.8.4
+beta: false
+enterprise: false
+---
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Konvoy&reg; version 1.8.4 was released on **
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
+
+### Supported versions
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**  | 1.18.x  |
+| **Maximum**  | 1.20.x  |
+| **Default**  | 1.20.11  |
+
+### New features and capabilities
+
+#### Networking
+
+- Calico has been updated to v3.20.2 to address an issue that prevented container deletion. (COPS-7092)
+
+#### Autoscaling
+
+- Fixed an issue installing the auto-provisioning component in an Ubuntu environment. (COPS-7102)
+
+#### Air-Gapped
+
+- Rhel8 FIPS packages have been updated to produce the necessary SHA256 signatures during installation. (COPS-7066)
+- Users within the system:authenticated group are now able to access group-appropriate URLs. (COPS-7067)
+
+### Bug Fixes
+
+#### Prometheus
+
+-  Prometheus addon upgrades now appropriately upgrade the alertmanager CRD.
+
+### Component versions
+
+- Ansible 2.9.16.0
+- Calico 3.20.2
+- Cluster-autoscaler v0.5.0
+- Containerd v1.4.7
+- Docker v19.03.15
+- kubeaddons-dispatch stable-1.20-1.4.6
+- kubeaddons-kommander stable-1.20-1.4.3
+- kubernetes-base-addons stable-1.20-4.3.0
+- Kubernetes v1.20.13
+- Kubeaddons v0.26.0
+- Mitogen a60c6c14a2473c895162a1b58a81bad0e63d1718
+- Terraform v0.13.7
+- Terraform AWS plugin ~> 3.0
+- Terraform Azure plugin ~> 2.31
+- Terraform GCP plugin ~> 3.42
+
+
+## Additional resources
+
+For information about working with native Kubernetes, see the [Kubernetes documentation](https://kubernetes.io/docs/home/).
+
+[kubernetes-doc](https://kubernetes.io/docs/home/)
+[konvoy-vsphere](/../../install/install-vsphere)
+[konvoy-ccontrol-plane-certificates](/../../networking/control-plane-certificates/)
+[security-groups-configuration](/../../install/install-aws/advanced-provisioning#security-groups)
```

Comment:
This seems to be a random set of links that have no bearing on the rest of the content. I believe these can be removed again.

Reply:
Agreed. You removed these links from the release notes we wrote and committed to the konvoy repo: https://github.com/mesosphere/konvoy/pull/3407/files#diff-82e4aa05feb6b6b145af5a0d46057039a2557a0c25afbeeb97e6103a30c0fd99

Reply:
I've gone ahead and removed these items, thanks @joejulian!